new/PRS14567Dastaw.xml

```diff
@@ -50,11 +50,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     
                     <persName xml:lang="gez" xml:id="n2" type="baptismal">ገብረ፡ ሕይወት፡</persName>
                     <persName xml:lang="gez" corresp="#n2" type="normalized">Gabra Ḥəywat</persName>
+                                                                             
+                    <persName xml:lang="gez" xml:id="n3" type="patronymic">ደምፀ፡</persName>
+                    <persName xml:lang="gez" corresp="#n3" type="normalized">Damḍa</persName>
```

Comment:
this is not what I meant ) I actually meant the normal way, as we usually refer to people
```
 <persName xml:lang="gez" xml:id="n1">ደስታው፡ ደምፀ፡</persName>
                    <persName xml:lang="gez" type="normalized" corresp="#n1">Dastāw Damḍa</persName>
```